landoui/static/src/js/pages/RevisionPage.js

```diff
@@ -0,0 +1,33 @@
+/* Simple class to manage events and state on the /revision/{id} page */
+Lando.RevisionPage = class {
```

Comment:
Got it I'll update this to use the pattern for the whole page e.g.`$page.landoRevisionPage()`.
Also doing so will probably remove the need for the global Lando variable I made (maybe it wouldn't hurt to keep it)